eng/SourceBuild.props

```diff
@@ -11,7 +11,9 @@
     The build script passes in the full path of the sln to build.  This must be overridden in order to build
     the cloned source in the inner build.
   -->
-  <Target Name="ConfigureInnerBuildArg" BeforeTargets="GetSourceBuildCommandConfiguration">
+  <Target Name="ConfigureInnerBuildArg"
+          BeforeTargets="GetSourceBuildCommandConfiguration"
+          Condition="'$(ArcadeBuildFromSource)' == 'true' or '$(DotNetBuildSourceOnly)' == 'true'">
```

Comment:
Shouldn't this entire props file be conditionally imported based on this condition instead?

Reply:
You mean SourceBuild.props? No, we always import that, unconditionally. There's no hook today that would allow that.